pom.xml

```diff
@@ -2006,18 +2006,6 @@
             <url>https://maven.codenvycorp.com/content/repositories/codenvy-public-snapshots/</url>
```

Comment:
note we will still have this because we have some artifacts that exists only on codenvy nexus, but once we will have it on central we will get rid of it.